core/src/test/java/io/questdb/test/griffin/AlterTableDetachPartitionTest.java

```diff
@@ -1191,19 +1191,9 @@ public void testDetachAttachSplitPartition() throws Exception {
                             FilesFacade ff = configuration.getFilesFacade();
                             Assert.assertTrue(ff.exists(path.$()));
 
-                            try {
-                                compile("ALTER TABLE " + tableName + " DETACH PARTITION LIST '2022-06-01'", sqlExecutionContext);
-                            } catch (CairoException e) {
-                                TestUtils.assertEquals("could not detach partition [table=testDetachAttachSplitPartition, " +
-                                        "detachStatus=DETACH_ERR_CANNOT_SQUASH, partitionTimestamp=2022-06-01T00:00:00.000Z, partitionBy=DAY]", e.getFlyweightMessage());
```

Comment:
nit: would be great to keep some tests around the DETACH_ERR_CANNOT_SQUASH status.

Reply:
Makes sense. Left a comment on the `detachPartition()` method.